build.gradle

```diff
@@ -974,8 +974,8 @@ project(':core') {
     from(project(':streams:streams-scala').configurations.runtime) { into("libs/") }
     from(project(':streams:test-utils').jar) { into("libs/") }
     from(project(':streams:test-utils').configurations.runtime) { into("libs/") }
-    from(project(':streams:examples').jar) { into("libs/") }
-    from(project(':streams:examples').configurations.runtime) { into("libs/") }
+    from(project(':streams-examples').jar) { into("libs/") }
+    from(project(':streams-examples').configurations.runtime) { into("libs/") }
```

Comment:
Why did we change the name?

Reply:
We move the sub-module `stream/examples` into it's own module `streams-examples`. I would be weird to only name it `examples` as there is not parent `streams` reference any longer.

Reply:
Is there an advantage to having it as a top level module instead?

Reply:
As describe on the Jira, the goal is to get rid of the dependency of `stream` model on `connect` module. It's ok for `streams-examples` to depend on it, but for `streams` is cleaner to not have it, and it's only there to pull in stuff for the examples.

Reply:
That seems independent of whether it's a submodule or not.

Reply:
Interesting. \cc @ableegoldman 

I actually just tried to only remove the `connect` dependency from `streams` module (and add the Jackson dependency we need fo `StateDirectory.java`) and it seems to work.

So maybe we can just keep sub-module `:streams:examples` as-is?

Reply:
Huh, maybe we're not even using `connect` in the examples to begin with (or at least, not anymore)? That actually doesn't sound too surprising, I've never seen Connect being used in any streams stuff I've looked at

> That seems independent of whether it's a submodule or not

@ijuma WDYM? If the examples are a submodule of `streams`, and the examples depend on `connect`, then doesn't that mean the `streams` module will pull in `connect`? Even if we may turn out not to depend on `connect` to begin with...)

Reply:
@ableegoldman No. I removed the `connect-json` dependency from `streams` and left it in `streams:examples` and there is no longer a `streams` dependency:

> compileClasspath - Compile classpath for source set 'main'.
> +--- project :clients
> +--- org.slf4j:slf4j-api:1.7.30
> +--- org.rocksdb:rocksdbjni:5.18.4
> +--- com.fasterxml.jackson.core:jackson-annotations:2.10.5
> \--- com.fasterxml.jackson.core:jackson-databind:2.10.5.1
>      +--- com.fasterxml.jackson.core:jackson-annotations:2.10.5
>      \--- com.fasterxml.jackson.core:jackson-core:2.10.5
>

Reply:
I think that's all you need to do.

Reply:
Sounds good. I will send a PR later this week fixing it to be as discussed here then. Probably even tonight (CET).